### PR TITLE
FE- calendar #8

### LINKED ISSUE #8: FE: Calendar
Create a calendar view for all our boards cards. Consider the user having multiple boards and how we can display where each card lives.

Note: Only use mock data right now as it's a front-end ticket, grab the board data from the context. 

\- Use a good library for calendar views
\- Add the calendar route and component.
\- Clicking on a card should display their details
\- Cards can be dragged to different dates, which will change their deadline
\-	Calendar should only show cards for the current board

Recommended library: https://github.com/jquense/react-big-calendar

### PR DESCRIPTION
**Pull request by uche-okoroafor**
### What this PR does (required):
-  Closes #8 
- Displays a big Calendar that contains cards with titles, tag color, and deadlines.
- The cards can be dragged to change the deadline.
### Screenshots / Videos (front-end only):
- Post at least one screenshot to show the feature in action. **Only required in front-end work**

https://user-images.githubusercontent.com/71725892/140648535-b807cc09-9b36-4bc7-bf63-86c4a16fcaee.mp4

### Any information needed to test this feature (required):
Run the messenger app,login and go to http://localhost:3000/calendar

### CODE REVIEW

File: server/controllers/dist/auth.dev.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+          res.status(400);
+          throw new Error("Invalid user data");
+
+        case 25:
+        case "end":
+          return _context.stop();
+      }
+    }
+  });
+}); // @route POST /auth/login
```
Comment (ezeilo-su) [inline server/controllers/dist/auth.dev.js:113]: Break this documentation into the next line

File: server/controllers/dist/auth.dev.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+          res.status(401);
+          throw new Error("Invalid email or password");
+
+        case 18:
+        case "end":
+          return _context2.stop();
+      }
+    }
+  });
+}); // @route GET /auth/user
```
Comment (ezeilo-su) [inline server/controllers/dist/auth.dev.js:183]: The same thin here as the previous comment

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Nice work on this one :+1:  Just go through the few comments I left and address them.

File: package.json [hunk 3]
```diff
@@ -6,5 +6,9 @@
   },
   "scripts": {
     "prepare": "husky install"
+  },
+  "dependencies": {
+    "date-fns": "^2.25.0",
+    "react-big-calendar": "^0.38.0"
```
Comment (moffatethan) [inline package.json:12]: Is this installed in the root directory? It should be installed in the client package file.

**Review by moffatethan — CHANGES_REQUESTED** [review]

**Comment by moffatethan** [comment]
Just that change there and the merge conflicts please. 👍🏻

**Review by moffatethan — APPROVED** [review]

**Pull request merged**